### DISCUSSION TITLE
Render faster

### DISCUSSION THREAD

**Discussion by Airioo** [bnpr/Malt#81] [Q&A] [answered]
Hello ! 
I'm wondering if there is tips to render faster ? Or make a viewport render image ? 
While I try viewport render image I only have a grey screen, and no object :(.

**Comment by pragma37** [comment, marked as answer]
Hi!

Most performance related settings are in the scene settings. They're documented here:
https://malt3d.com/#/docs/NPR-Pipeline?id=scene

The values you should lower depends on your specific scene.
Use the lowest Sampling Grid Size you can as long as your render doesn't look noisy.
Use the lowest Transparency Layers you can. If you are not using transparent materials you can set Transparency Layers to 0.
Lower your Sun Shadows Max Distance as much as your scene allows.
And lower your shadowmaps resolution as much as you can as long as you are happy with the resulting quality.

Keep in mind that Malt can override each parameter for Preview (Material Preview viewport mode) and Final Render (F12).

Viewport Render Image/Animation is not supported and as far as I know there's no way to support it from external renderers.

**Reply by Airioo** [reply]
Ok nice thanks !
